spec.bs

```diff
@@ -3620,7 +3627,9 @@ The <dfn for="interest group">estimated size</dfn> of an [=interest group=] |ig|
   [=interest group/owner=].
 1. The [=string/length=] of |ig|'s [=interest group/name=].
 1. 8, which is the size of |ig|'s [=interest group/priority=].
-1. The [=string/length=] of |ig|'s [=interest group/execution mode=].
+1. 4, which is the size of |ig|'s [=interest group/execution mode=].
+
+  Note: Each of [=interest group/execution mode=]'s value reprensents a {{long}} integer, so 4 bytes.

```

Comment:
(based on https://github.com/WICG/turtledove/pull/954#issuecomment-1902271691)
Is this reasonable, or I add this sentence for the definition of [=interest group/execution mode=]?

Reply:
This is a normative requirement so you'll have to specify this in the definition of the new enum, saying that each of its members represent a `{{long]}`. This cannot be in `Note:` form either. However, it wouldn't be bad to keep this note here just for extra documentation... your call on that one.